### ISSUE TITLE
Switch lang-picker-toggler to a button

### ISSUE THREAD

**Issue by XhmikosR** [nodejs/nodejs.org#2545]
Should be better for accessibility and we will probably simplify a few things.

I'll take a stab at it in the next days.

**Comment by moshfeu** [comment]
If you already touch it, it will be better to add more accessibility aspects to the list (options also should be focusable, `aria-expanded`, `aria-haspopup` etc.)

**Comment by XhmikosR** [comment]
See https://github.com/nodejs/nodejs.org/pull/2566

Only toggling aria-expanded is left.

**Comment by moshfeu** [comment]
🔥
I'm not familiar with that syntax but I can't see how the `option`s will be focusable (navigate via tabs, or even arrows)

**Comment by XhmikosR** [comment]
That doesn't work, unfortunately. Nor the toggler closes with the Esc key. But you are more than welcome to help me out in that PR :)

**Comment by moshfeu** [comment]
Ok.
I'll see what I can do :)
